usaspending_api/references/management/commands/load_offices.py

```diff
@@ -44,6 +44,15 @@ def process_data(self):
         logger.info(f"Loaded: {new_rec_count:,} records")
         logger.info("Committing transaction to database")
 
+        logger.info("Deleting all existing offices that are not linked to a transaction")
+        # Identify offices that do not correspond to any transactions, using only USAS DB
+        with connections[DEFAULT_DB_ALIAS].cursor() as cursor:
+            cursor.execute(self.usas_unlinked_offices_sql)
+            office_values = dictfetchall(broker_cursor)
+            for office_code in office_values:
+                print(office_code)
+                Office.objects.filter(office_code=office_code["office_code"]).delete()
```

Comment:
Is there a way we could do this as a single SQL query rather than one query per deleted office?

Reply:
Yes change pushed